### PR TITLE
Icon component

### PR DESCRIPTION
**Pull request by samuelg0rd0n**

### CODE REVIEW

**Comment by FilipPyrek** [comment]
@samuelg0rd0n why isn't this commit using conventional commits format? 
https://github.com/purple-technology/phoenix-components/pull/164/commits/0b5c9c9981806fb9b97a30e892376496dbcbe1d4

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		"prepare": "husky install",
 		"prebuild": "rimraf dist",
 		"dev": "npm run storybook",
-		"storybook": "start-storybook -p 6006 --no-manager-cache",
```
Comment (FilipPyrek) [inline package.json:37]: Why is this flag being removed?

**Comment by samuelg0rd0n** [comment]
Interesting, I don't know. Well, I must have forgotten. But more importantly, I thought we had some kind of pre-commit hook to prevent this, don't we?

**Review by FilipPyrek — APPROVED** [review]

**Comment by FilipPyrek** [comment]
> Interesting, I don't know. Well, I must have forgotten. But more importantly, I thought we had some kind of pre-commit hook to prevent this, don't we?

Well, there should be one. I can see that the commit is quite old.. so most probably it was in the times of broken husky - which was fixed in #162 

I leave it up to you if you want to do a rebase and rename the commit or if you want to keep it like this for now. 👍

File: package.json (hunk 1 above)
Comment (samuelg0rd0n) [reply package.json:37]: From the docs

> The flag --no-manager-cache disables the internal caching of Storybook and can severely impact your Storybook loading time, so only use it when you need to refresh Storybook's UI, such as when editing themes.

I put it there sometime ago when I had problems with Storybook not updating but I think it wasn't even the solution, I forgot it there and committed. But there's no reason for this flag to be there.

Comment (FilipPyrek) [reply package.json:37]: 🔝 👍

**Pull request merged**